### PR TITLE
Add ENR bitfield

### PR DESCRIPTION
**Pull request by pawanjay176**
Add support for attestation bitfield in ENR records.

### CODE REVIEW

File: misc/enr/src/lib.rs [hunk 1]
```diff
@@ -395,6 +400,11 @@ impl Enr {
         self.add_key("tcp6", tcp.to_be_bytes().to_vec(), keypair)
     }
 
+    /// Set attestation subnet bitfield.
+    pub fn set_attnets(&mut self, attnets: Vec<u8>, keypair: &Keypair) -> Result<bool, EnrError> {
```
Comment (AgeManning) [inline misc/enr/src/lib.rs]: I think it would be nicer to have attnets to be of type `&[u8]` to be more versatile when updating.

Comment (pawanjay176) [reply misc/enr/src/lib.rs]: Makes sense. Will make the changes.

**Comment by pawanjay176** [comment]
@AgeManning Made the changes. Also merged latest master to fix the protobuf issue.

**Comment by AgeManning** [comment]
I think you need to target the discv5 branch for this PR

**Comment by pawanjay176** [comment]
@AgeManning messed up reverting the merge. Had to force push. Should be fine now.

**Pull request merged**